test_settings.py

```diff
@@ -16,3 +16,11 @@
         'cms.test_utils.project.sampleapp'
     ],
 }
+
+
+def run():
```

Comment:
Two reasons:

 * If you add a runner you can provide default options (like ``--cms`` etc) (see https://github.com/aldryn/aldryn-blueprint/pull/11/files#diff-d62ed1b8ca56af3ad3de9c9e4fa584beR110)
* The runner enables to leverage the ``test_suite`` argument in setup.py (https://github.com/aldryn/aldryn-blueprint/pull/11/files#diff-2eeaed663bd0d25b7e608891384b7298R37) so running tests is nothing more that ``python setup.py test``